RAM_KUMAR/assignment/Calculator/calculator.py

```diff
@@ -0,0 +1,13 @@
+class Calculator :
+    def add(self, a, b):
+        return float(a) + float(b)
+
+    def subtract(self, a , b):
+        return float(a) - float(b)
+
+    def multiply(self, a , b):
+        return float(a) * float(b)
+
+    def divide(self, a , b):
+        return float(a) / float(b)
```

Comment:
Variable names should not be used as "a" or "b" give some meaningful name. 
Ex-  num1, num2